layouts/examples/single.html

```diff
@@ -1,19 +1,14 @@
 {{ define "main" }}
 <h3>{{ .Title }}</h3>
 {{ .Content }}
-
 <br>
+<hr>
 <details>
-    <summary><i>try it out</i></summary>
-
-    install the cairo toolchain if you haven't yet:
-    <pre>
-	curl -L https://github.com/franalgaba/cairo-installer/raw/main/bin/cairo-installer | bash
-        </pre>
-
-    then checkout and run the example:
-    <pre>
-        </pre>
+    <summary><i>How to install the toolchain</i></summary>
```

Comment:
I'd keep the "try it out", and add this as a section header. Example:

<details>
<summary><i>try it out!</i></summary>

<h4> 1. Install the toolchain </h4>

...

</details>

Reply:
I considered that but there are some examples which already include a little "run with this command" paragraph, and generally the command depends on the example, so maybe that's not the best idea unless the run step in your list could be changed in each .md, somehow

Reply:
The thing with the "run with this command" paragraph is that not every example has it, and it isn't a complete set of instructions. Doing it for every example, and adding instructions for dependency installation would result in a lot of boilerplate. That's why I think we should move those to the "try it out" section we have now.

To simplify that, we should try to standardize the way examples are run. I've noticed there are mainly two ways to run the examples: with `cairo-run` and with `cairo-test`. I found this could be done by adding a parameter, let's say, `example_type` with values `main` and `test`, and then modify the "run this command" part to:

```
{{ if eq .Params.example_type "main" }}
  <div>cairo-run ...</div>
{{ else if eq .Params.example_type "test" }}
  <div>cairo-test ...</div>
{{ end }}
```

The instructions also can be made a bit more general, like: "paste the example into a file named `test.cairo`", and "run `cairo-test test.cairo`".

Reply:
That sounds good! will implement. Thank you!